components/automate-ui/src/app/pages/user-management/user-form/user-form.component.html

```diff
@@ -4,33 +4,38 @@
       <span class="label">User's Full Name</span>
       <input chefInput formControlName="fullname" type="text">
     </label>
-    <chef-error *ngIf="createUserForm.get('fullname').hasError('required') && createUserForm.get('fullname').dirty">Full
-      name required</chef-error>
+    <chef-error *ngIf="(createUserForm.get('fullname').hasError('required') || createUserForm.get('fullname').hasError('pattern')) && createUserForm.get('fullname').dirty">
+      Full name is required
+    </chef-error>
   </chef-form-field>
   <chef-form-field>
     <label>
       <span class="label">Username</span>
       <input chefInput formControlName="username" type="text">
     </label>
-    <chef-error *ngIf="createUserForm.get('username').hasError('pattern') && createUserForm.get('username').dirty">Invalid
-      username</chef-error>
+    <chef-error *ngIf="(createUserForm.get('username').hasError('required') || createUserForm.get('username').hasError('pattern')) && createUserForm.get('username').dirty">
+      Invalid username
```

Comment:
can we be more specific here? this error isn't very helpful...

Reply:
Split required and pattern to allow giving two specific error messages. 👍